### PR TITLE
Add TextArea component and fix TextField

### PR DESCRIPTION
**Pull request by elvisvoer**

### CODE REVIEW

**Comment by elvisvoer** [comment]
Second commit closes airtame/airtame-gooey#38

File: src/components/TextArea/TextArea.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    return (
+      <div className={textAreaClassNames}>
+        <textarea
+          id={id}
+          className="gooey-text-area__input"
+          ref={textArea => this.textArea = textArea}
+          onChange={this.handleInputValue}
+          disabled={isDisabled}
+          readOnly={isReadOnly}
+          value={this.state.textAreaValue}
```
Comment (mmellado) [inline src/components/TextArea/TextArea.js]: This is a text area. The textarea should take the value inside the tag not as a value attribute:

```jsx
<textarea {...props}>{this.state.textAreaValue}</textarea>
```

File: src/components/TextArea/TextArea.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        'gooey-text-area--readonly': isReadOnly,
+      },
+      className
+    );
+
+    return (
+      <div className={textAreaClassNames}>
+        <textarea
+          id={id}
+          className="gooey-text-area__input"
```
Comment (mmellado) [inline src/components/TextArea/TextArea.js]: gooey-text-area__textarea

File: src/components/TextArea/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export default from './TextArea';
```
Comment (mmellado) [inline src/components/TextArea/index.js]: add empty line at the end of the file

File: test/js/components/TextArea.spec.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import TextArea from '../../../src/components/TextArea';
+
+describe('<TextArea />', () => {
+  it('should render the textarea field', () => {
+    const wrapper = render(<TextArea />);
+    expect(wrapper).to.have.tagName('div');
+  });
+
+  it('should create a textarea field with an initial value', () => {
```
Comment (mmellado) [inline test/js/components/TextArea.spec.js]: I'm surprised this test passes given the value is being passed as an attribute

**Pull request merged**